tests/system/genesis_io.go

```diff
@@ -31,3 +31,15 @@ func GetGenesisBalance(rawGenesis []byte, addr string) sdk.Coins {
 	}
 	return r
 }
+
+// SetCodeUploadPermission sets the code upload permissions
+func SetCodeUploadPermission(t *testing.T, permission string, addresses []string) GenesisMutator {
```

Comment:
personal preference :-)
```suggestion
func SetCodeUploadPermission(t *testing.T, permission string, addresses ...string) GenesisMutator {
```